Canvas/SubmitAssignment/SubmitAssignmentPresenter.swift

```diff
@@ -67,10 +71,26 @@ class SubmitAssignmentPresenter {
         env = AppEnvironment.shared
         env.userDidLogin(session: session)
         sharedContainer = container
+
+        if let courseID = env.userDefaults?.submitAssignmentCourseID {
```

Comment:
when does this get set?

Reply:
https://github.com/instructure/canvas-ios/pull/436/files#diff-a9a597152bf01b148f87e0546fc56aceR41

Reply:
This is when they are annotating a file in our app and then want to submit their annotations using our submit share extension.